packages/contracts-ops/contracts/Config.sol

```diff
@@ -35,6 +35,10 @@ abstract contract Config is INomadProtocol {
         config = vm.readFile(file);
     }
 
+    function __Config_reload(string memory file) internal {
```

Comment:
should not be named as initializer. `reload()` is sufficient

Reply:
fixed + a bunch of warning fixing in 5c1e4c64

config now stores its path, and reloads from it
inheritors now use `configPath` instead of passing it as an arg

Reply:
https://github.com/nomad-xyz/monorepo/pull/511/commits/5c1e4c642bf94433c06efc2059a0e65608d756bd